app/src/main/java/com/example/filemanager/AllFilesViewModel.kt

```diff
@@ -1,21 +1,32 @@
 package com.example.filemanager
 
 import android.os.Environment
-import android.util.Log
 import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
-import java.io.File
 
 class AllFilesViewModel() : ViewModel() {
 
     val files = MutableLiveData<List<ListModel>>()
+    private val listRootDirectoryFiles: ListModel = ListModel(Environment.getExternalStorageDirectory())
```

Comment:
It is a wrong name for a field, field names should not express an action.

Reply:
it was named as rootDirectory and you said to rename it to listRootDirectoryFiles